src/Reflection/AST/Show.agda

```diff
@@ -58,6 +59,10 @@ showVisibility visible   = "visible"
 showVisibility hidden    = "hidden"
 showVisibility instance′ = "instance"
 
+showQuantity : Quantity → String
+showQuantity quantity-0 = "quantity-0"
+showQuantity quantity-ω = "quantity-ω"
```

Comment:
I realise I haven't kept up with the PRs on `experimental`, so I assume that these are the agreed `String` renderings... but they maybe seem a bit verbose?

Reply:
Happy to leave them as-is... the style for `show` operations does seem to favour a more verbose=informative style.